website/package.json

```diff
@@ -1,14 +1,23 @@
 {
   "scripts": {
     "examples": "docusaurus-examples",
-    "start": "docusaurus-start",
-    "build": "docusaurus-build",
+		"clean:docs": "rm -r ../docs/sync/*",
```

Comment:
Needs to be `rm -rf ../docs/sync/*`

Reply:
Oh great! Yeah I would just add a placeholder file in there for `rm -r` to remove, but `-rf` seems to work too

Reply:
Just pushed that to the PR